data_sources/cisco_secure_firewall_threat_defense_connection_event.yml

```diff
@@ -0,0 +1,117 @@
+name: Cisco Secure Firewall Threat Defense Connection Event
+id: 18878597-8f8a-4bca-a805-bfbe35e00032
+version: 1
+date: '2025-04-01'
+author: Nasreddine Bencherchali, Splunk
+description: Data source object for raw connection events from Cisco Secure Firewall Threat Defense
+source: not_applicable
+sourcetype: cisco:sfw:estreamer
+supported_TA:
+- name: Cisco Security Cloud
+  url: https://splunkbase.splunk.com/app/7404
+  version: 3.1.1
+fields:
+- AC_RuleAction
+- action
+- app
+- Application
+- bytes_in
+- bytes_out
+- ClientAppDetector
+- ClientApplication
+- connection_id
+- ConnectionDuration
+- ConnectionID
+- date_hour
+- date_mday
+- date_minute
+- date_month
+- date_second
+- date_wday
+- date_year
+- date_zone
+- dest
+- dest_interface
+- dest_ip
+- dest_port
+- dest_zone
+- device_id
+- DeviceUUID
+- dvc
+- EgressInterface
+- EgressVRF
+- EgressZone
+- EVE_Fingerprint
+- EVE_Process
+- EVE_ProcessConfidencePct
+- EVE_ThreatConfidenceIndex
+- EVE_ThreatConfidencePct
+- eventtype
+- EventType
+- FirewallPolicy
+- FirewallRule
+- FirstPacketSecond
+- host
+- index
+- IngressInterface
+- IngressVRF
+- IngressZone
+- InitiatorBytes
+- InitiatorIP
+- InitiatorPackets
+- InitiatorPort
+- instance_id
+- InstanceID
+- LastPacketSecond
+- linecount
+- NAP_Policy
+- NAT_InitiatorIP
+- NAT_InitiatorPort
+- NAT_ResponderIP
+- NAT_ResponderPort
+- packets_in
+- packets_out
+- PrefilterPolicy
+- Protocol
+- punct
+- ResponderBytes
+- ResponderIP
+- ResponderPackets
+- ResponderPort
+- rule
+- source
+- sourcetype
+- splunk_server
+- src_interface
+- src_ip
+- src_port
+- src_zone
+- SSL_ActualAction
+- SSL_CertFingerprint
+- SSL_CipherSuite
+- SSL_ExpectedAction
+- SSL_FlowStatus
+- ssl_hash
+- ssl_policies
+- SSL_Policy
+- SSL_ServerCertStatus
+- ssl_signature_algorithm
+- ssl_version
+- SSL_Version
+- tag
+- tag::eventtype
+- timeendpos
+- timestartpos
+- transport
+- url
+- URL
+- vendor_product
+- WebApplication
+output_fields:
```

Comment:
i think for these network events,  adding action as a mandatory output field would be good since that directly tells the analyst what was the action at the end of the day for any alert from these detections

Reply:
Added action to all related rules and as a required output to the datasource